addon-test-support/load.js

```diff
@@ -19,4 +22,60 @@ export default function loadEmberExam() {
   if (window.Testem) {
     patchTestemOutput(TestLoader);
   }
+
+  testLoader = new TestLoader();
 }
+
+// loadTests() is equivalent to ember-qunit's loadTest() except this does not create a new TestLoader instance
+function loadTests() {
+  testLoader.loadModules();
+}
+
+// setupQUnitCallBacks() registers QUnit callbacks neeeded for the load-balance option.
+function setupQUnitCallbacks(qunit) {
+  if (!location.search.includes("_loadBalance")) {
+    return;
+  }
+  qunit.begin(function() {
+    return new Promise(function(resolve) {
+      Testem.once('next-module-response', function getTestModule(moduleName) {
+        if (moduleName !== undefined) {
+          loadIndividualModule(moduleName);
+        }
+        Testem.removeEventCallbacks('next-module-response', getTestModule);
+        resolve();
+      });
+      Testem.emit('get-next-module');
+    });
+  });
+  qunit.moduleDone(function() {
```

Comment:
similar thing here.
one difference, we can probably do without `resolveCallbackFn` and use resolve directly.
```
qunit.moduleDone(function() {
    return new Promise(function(resolve) {
      Testem.on('next-module-response', function getTestModule(moduleName) {
        if (moduleName !== undefined) {
          loadIndividualModule(moduleName);
        }

        // if no tests were added, request the next module
        if (QUnit.config.queue.length === 0) {
          Testem.emit('get-next-module');
        } else {
          Testem.removeDuplicateCallbacks('next-module-response', getTestModule);
          Testem.removeDuplicateCallbacks('module-queue-complete', resolve);
          resolve();
        }
      });
      Testem.on('module-queue-complete', resolve);
      Testem.emit('get-next-module');
    });
  });
```